### PR TITLE
Add support for dynamic loading/unloading

### PR DESCRIPTION
**Pull request by rsbondi**
#16 @darosior if you could check the readme if that description makes sense, I noticed that any plugin loaded via the CLI can be stopped regardless of how I set this flag, seems logical.

Also `max_accepted_htlcs` typo fixed

### CODE REVIEW

**Comment by darosior** [comment]
You might want to default to dynamic. Static plugin is the special case (at least that's what we did for Pylightning)

File: glightning/plugin.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ type OpenChannelEvent struct {
 	HtlcMinimumMillisatoshis          string `json:"htlc_minimum_msat"`
 	//FeeratePerKw ..//??
 	ToSelfDelay      uint16 `json:"to_self_delay"`
-	MaxAcceptedHtlcs uint16 `json"max_accepted_htlcs"`
```
Comment (niftynei) [inline glightning/plugin.go]: while a valid fix, this change is not related to the rest of the commit. can you pull it out into a separate PR/commit?

Comment (rsbondi) [reply glightning/plugin.go]: I removed and pushed, build fail, I will make a separate PR

**Comment by niftynei** [comment]
here's the error from the CI. it's running `go test -short ./...`

```
    plugin_test.go:209: 
        	Error Trace:	plugin_test.go:209
        	            				plugin_test.go:148
        	Error:      	Not equal: 
        	            	expected: "{\"jsonrpc\":\"2.0\",\"result\":{\"options\":[],\"rpcmethods\":[],\"hooks\":[\"db_write\",\"peer_connected\",\"invoice_payment\",\"openchannel\",\"htlc_accepted\"]},\"id\":\"aloha\"}"
        	            	actual  : "{\"jsonrpc\":\"2.0\",\"result\":{\"options\":[],\"rpcmethods\":[],\"hooks\":[\"db_write\",\"peer_connected\",\"invoice_payment\",\"openchannel\",\"htlc_accepted\"],\"dynamic\":true},\"id\":\"aloha\"}"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-{"jsonrpc":"2.0","result":{"options":[],"rpcmethods":[],"hooks":["db_write","peer_connected","invoice_payment","openchannel","htlc_accepted"]},"id":"aloha"}
        	            	+{"jsonrpc":"2.0","result":{"options":[],"rpcmethods":[],"hooks":["db_write","peer_connected","invoice_payment","openchannel","htlc_accepted"],"dynamic":true},"id":"aloha"}
        	Test:       	TestManifestWithHooks
```
